packages/fbtee/scripts/rewrite-filepaths.ts

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env node --experimental-strip-types --no-warnings
+import { readFileSync, writeFileSync } from 'node:fs';
+import { join } from 'node:path';
+
+const fileName = join(import.meta.dirname, '../Strings.json');
+const strings = JSON.parse(readFileSync(fileName, 'utf8'));
+
+for (const key in strings.phrases) {
+  strings.phrases[key].filepath = 'node_modules/fbtee/Strings.json';
+}
+
+writeFileSync(fileName, JSON.stringify(strings, null, 2), 'utf8');
```

Comment:
The strings are exported with their source filepath which doesn't exist in a downstream project. This replaces the file paths with the actual location `node_modules/fbtee/lib/index.js`.